topostats/tracing/nodestats.py

```diff
@@ -3,17 +3,21 @@
 from __future__ import annotations
 
 import logging
-import math
 
 import networkx as nx
 import numpy as np
 import numpy.typing as npt
+from numpy.typing import NDArray
```

Comment:
Essentially redundant given line 9.

I'm gradually favouring and getting in the habit of importing at the module level as it makes it easier to know what package modules belong to. In this instance not much of an issue but there may be other cases where there is the same function from different packages and by importing at the module level it makes it easier to distinguish.

A Find and Replace across the file for `NDArray` > `npt.NDArray` and removing this line should solve things.

Reply:
Done now, AFAIK I've found all of them.